qiskit_alice_bob_provider/backend.py

```diff
@@ -24,17 +24,16 @@
 from .api import jobs
 from .api.client import ApiClient
 from .ensure_preparation_pass import EnsurePreparationPass
-from .job import AliceBobJob
+from .job import AliceBobRemoteJob
 from .qir_to_qiskit import ab_target_to_qiskit_target
 from .utils import camel_to_snake_case, snake_to_camel_case
 
 
-class AliceBobBackend(BackendV2):
-    """Class representing the single cat simulator target accessible via the
-    Alice & Bob API"""
+class AliceBobRemoteBackend(BackendV2):
+    """Class representing the targets accessible via the Alice & Bob API."""
```

Comment:
Updating the docstring in a Driveby. The targets also would include multiple cats and actual hardwares, not only simulators.

Reply:
Great thanks!